packages/backend/src/panels/AbstractWebviewPanel.ts

```diff
@@ -111,34 +99,18 @@ export abstract class AbstractWebviewPanel {
     }
   }
 
-  protected async initHtmlContent() {
-    let indexHtml = await fsextra.readFile(
-      path.join(this.mediaPath, this.htmlFileName),
-      "utf8"
-    );
+  protected initHtmlContent(): void {
+    let indexHtml = readFileSync(path.join(this.mediaPath, this.htmlFileName), "utf8");
     if (indexHtml) {
       // Local path to main script run in the webview
-      const scriptPathOnDisk = vscode.Uri.file(
-        path.join(this.mediaPath, path.sep)
-      );
-      const scriptUri = this.webViewPanel.webview.asWebviewUri(
-        scriptPathOnDisk
-      );
+      const scriptPathOnDisk = vscode.Uri.file(path.join(this.mediaPath, path.sep));
```

Comment:
is it should end with '/' ?

Reply:
yes, it will not open panel without this